app/stats/views.py

```diff
@@ -100,35 +110,49 @@ def retrieve_annual_chart(from_year = 2009, to_year = date.today().year):
         fontSize = 18,
     )
 
-def retrieve_monthly_chart(from_year = 2017, to_year = date.today().year):
-    chart_width = 1000
```

Comment:
where ever you have numbers with more than 4 digits `1000`, or `1000000`,  please use underscores to split them, so 1000000 becomes 1_000_000. It's much easier to read!